crates/protocol/src/info/variant.rs

```diff
@@ -95,9 +98,26 @@ impl L1BlockInfoTx {
                 blob_base_fee: l1_header.blob_fee(BlobParams::cancun()).unwrap_or(1),
                 blob_base_fee_scalar,
                 base_fee_scalar,
-            }))
-        } else {
-            Ok(Self::Ecotone(L1BlockInfoEcotone {
+            }));
+        }
+
+        if rollup_config.is_isthmus_active(l2_block_time)
+            && rollup_config.isthmus_time.unwrap_or_default() != l2_block_time
+        {
+            let (operator_fee_scalar, operator_fee_constant) = if scalar[0]
+                == L1BlockInfoIsthmus::L1_SCALAR
+            {
+                let operator_fee_scalar = Ok::<u32, BlockInfoError>(u32::from_be_bytes(
+                    scalar[12..16].try_into().map_err(|_| BlockInfoError::OperatorFeeScalar)?,
```

Comment:
I think this is incorrect:

Operator fee params are not included in the `FEE_SCALARS` `ConfigUpdate`, but in a new one: `OPERATOR_FEE_PARAMS`

More info here: https://github.com/ethereum-optimism/specs/pull/382/files#diff-7597cdda46cdabce58537c40594329d56c0230b49ef4a126da0932f0a1992272R37

Reply:
Ah this had changed last time i reviewed the specs - my mistake. I've updated to include the operator fees as part of the system config now so it can be parsed in the config update.